config/config.exs

```diff
@@ -11,16 +11,16 @@ config :nerves_hub_cli,
 
 # API HTTP connection.
 config :nerves_hub_core,
-  host: "0.0.0.0",
-  port: 4002
+  api_host: "0.0.0.0",
+  api_port: 4002
 
 # Device Websocket/Channel connection.
 config :nerves_hub, NervesHub.Socket, url: "wss://0.0.0.0:4001/socket/websocket"
 
 # Device HTTP connection.
-config :nerves_hub, NervesHub.HTTPClient,
-  host: "0.0.0.0",
-  port: 4001
+config :nerves_hub,
+  device_host: "0.0.0.0",
```

Comment:
Sigh, this make ms sad that the reference to the server hostname makes it seem like this is _the_ device's host / port

Reply:
Yes, I was thinking that as well. I'm finding the names of many parts of NervesHub to be pretty confusing. We may need to take a step back and reconsider the overall naming of things. At the low level, though, this name just needs to refer to `device.nerves-hub.org` and not be confused with `nerves-hub.org`, `api.nerves-hub.org`, or `www.nerves-hub.org`.